exporters/transform/flatson_transform.py

```diff
@@ -0,0 +1,38 @@
+from exporters.transform.base_transform import BaseTransform
+from flatson import Flatson
```

Comment:
this is an optional dependency and should be imported lazily only where it will be used (see odo writer for an example).